interview-questions.md

```diff
@@ -8,42 +8,46 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
You started out really good on the interview answers then it seemed to have lost the momentum. Ensure you are supporting your definitions with either examples or personal experience.